data/crac/crac-result-extensions/src/main/java/com/farao_community/farao/data/crac_result_extensions/CnecResult.java

```diff
@@ -28,6 +28,8 @@
     private double minThresholdInA;
     private double maxThresholdInA;
 
+    private double loopflow; //loopflow value
```

Comment:
Could you also add the unit in variable name (which is a good habit I think)

Reply:
updated